pkg/machinery/client/client.go

```diff
@@ -563,6 +563,11 @@ func (c *Client) LS(ctx context.Context, req *machineapi.ListRequest) (stream ma
 	return c.MachineClient.List(ctx, req)
 }
 
+// DU implements the proto.OSClient interface.
+func (c *Client) DU(ctx context.Context, req *machineapi.DiskUsageRequest) (stream machineapi.MachineService_DiskUsageClient, err error) {
```

Comment:
I know we set a bit example with `list` (function is named `LS`), but since this is new I think we should have the name match the `MachineClient` name (`DiskUsage`).